hugolib/author.go

```diff
@@ -13,23 +13,51 @@
 
 package hugolib
 
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cast"
+)
+
+var (
+	onlyNumbersRegExp = regexp.MustCompile("^[0-9]*$")
+)
+
 // AuthorList is a list of all authors and their metadata.
 type AuthorList map[string]Author
 
+// Authors returns a slice of authors sorted by ID
+func (al AuthorList) Authors() []Author {
```

Comment:
I can go ahead and do that. How would you like me to implement a sort if not using the stdlib interface?